charts/redpanda/values.go

```diff
@@ -0,0 +1,40 @@
+package redpanda
+
+type Values struct {
```

Comment:
I wonder if `Values` structure could generate our `values.yaml` definition with `values.schema.json`. Or vice versa to keep all places in sync.

Reply:
Absolutely! I'm not sure which direction the generation should go IE go -> yaml or yaml -> go? We'll want to ensure that documentation is generated appropriately for everything though.